recipes/mod_wsgi/yum_requirements.txt

```diff
@@ -0,0 +1,4 @@
+httpd-devel
+gcc
+glibc
+glibc-devel-static
```

Comment:
Do we need these or can they be dropped?

Reply:
if we don't need it for the build we can drop it.

Reply:
removed